prisma/createData.ts

```diff
@@ -0,0 +1,63 @@
+import type { PrismaClient } from "@prisma/client";
+import { Picsum } from "picsum-photos";
+
+export const createRole = async (prisma: PrismaClient, roleName: string) => {
+  const role = await prisma.userRole.upsert({
+    where: {
+      roleName,
+    },
+    update: {},
+    create: {
+      roleName,
+    },
+  });
+  return role;
+};
+
+export const createUser = async (
```

Comment:
I'm not the biggest fan of this function having 11 arguments... Do you think you could instead make it take an object with all of these arguments as keys? I saw this problem with this function when it had 5 arguments but 11 is just too many haha.

Reply:
You can leave it as is but if you want you can change it. It is non critical part of code and it's too bad as is but is a bit of a smell imho

Reply:
I can get to fixing that on Thursday!